mocks/src/memory_storage_provider.rs

```diff
@@ -11,19 +12,19 @@ use dlc_manager::contract::{
 use dlc_manager::sub_channel_manager::Action;
 use dlc_manager::subchannel::{SubChannel, SubChannelState};
 use dlc_manager::Storage;
-use dlc_manager::{error::Error as DaemonError, ChannelId, ContractId, Utxo};
+use dlc_manager::{error::Error as DaemonError, ContractId, Utxo};
 use secp256k1_zkp::{PublicKey, SecretKey};
 use simple_wallet::WalletStorage;
 use std::collections::HashMap;
 use std::sync::{Mutex, RwLock};
 
 pub struct MemoryStorage {
     contracts: RwLock<HashMap<ContractId, Contract>>,
-    channels: RwLock<HashMap<ChannelId, Channel>>,
-    sub_channels: RwLock<HashMap<ChannelId, SubChannel>>,
+    channels: RwLock<HashMap<[u8; 32], Channel>>,
```

Comment:
Is there a reason for not using the types here?

Reply:
Hehe, I had the types here until the very last second, when the compiler started giving me a hard time about the types not implementing certain traits. I can give it a go again.